src/Symfony/Component/Serializer/Exception/AggregatedException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+class AggregatedException extends RuntimeException implements AggregableExceptionInterface
+{
+    /**
+     * @var ExceptionInterface[]
+     */
+    private $exceptions;
+
+    public function __construct(array $exceptions, \Throwable $previous = null)
+    {
+        parent::__construct('The following arguments are invalid', 0, $previous);
```

Comment:
Since there is no way to know what went wrong maybe a more generic message:
```suggestion
        parent::__construct('Errors occurred during the denormalization process', 0, $previous);
```

Reply:
Done